.goreleaser.darwin.yaml

```diff
@@ -39,6 +42,23 @@ archives:
     files:
       - LICENSE
       - README.md
+  # tar.gz archives for Mac OS:
+  #   - will not be notarized
+  #   - will be installable via homebrew (i.e. "brew install kitops")
+  - id: kit-macos-targz-archive
+    format: tar.gz
+    builds:
+      - kit-macos
+    name_template: >-
+      {{ .ProjectName }}-
+      {{- tolower .Os }}-
+      {{- if eq .Arch "amd64" }}x86_64
+      {{- else if eq .Arch "386" }}i386
+      {{- else }}{{ .Arch }}{{ end }}
+      {{- if .Arm }}v{{ .Arm }}{{ end }}
```

Comment:
On macos we can probably simplify this, I don't think there's any i386 macs left :)

Reply:
I copied this section from the one above it where the .zip binaries are configured